pkg/daemon/ceph/client/pool.go

```diff
@@ -179,17 +185,20 @@ func givePoolAppTag(context *clusterd.Context, clusterName string, poolName stri
 	return nil
 }
 
-func CreateECPoolForApp(context *clusterd.Context, clusterName string, newPool CephStoragePoolDetails, appName string, enableECOverwrite bool) error {
+func CreateECPoolForApp(context *clusterd.Context, clusterName string, newPool CephStoragePoolDetails, appName string, enableECOverwrite bool, erasureCodedConfig model.ErasureCodedPoolConfig) error {
 	args := []string{"osd", "pool", "create", newPool.Name, strconv.Itoa(newPool.Number), "erasure", newPool.ErasureCodeProfile}
 
 	buf, err := ExecuteCephCommand(context, clusterName, args)
 	if err != nil {
 		return fmt.Errorf("failed to create EC pool %s. %+v", newPool.Name, err)
 	}
 
+	if err = SetPoolProperty(context, clusterName, newPool.Name, "min_size", strconv.FormatUint(uint64(erasureCodedConfig.DataChunkCount), 10)); err != nil {
+		return fmt.Errorf("failed to set min size to 1 for pool %s. %+v", newPool.Name, err)
```

Comment:
this error message hard codes "1" but we were actually trying to set it to the number of data chunks

Reply:
Fixed.